src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+interface ITextFieldProps {
+  customClassNames?: string;
+  value?: string;
+  label?: string;
+  type: string;
+  name?: string;
+  placeholder?: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+function TextField(props: ITextFieldProps) {
+  return (
+    <label>
+      {props.label}
```

Comment:
Since `props.label` could be undefined, why don't we conditionally render the <label></label>? You may have to make it a <label for> element in that case.